pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/defaultcolumn/BaseDefaultColumnHandler.java

```diff
@@ -372,6 +394,12 @@ protected boolean createColumnV1Indices(String column)
             return false;
           }
 
+          // TODO: Support forward index disabled derived column
+          if (_indexLoadingConfig.getForwardIndexDisabledColumns().contains(column)) {
+            LOGGER.warn("Skip creating forward index disabled derived column: {}", column);
```

Comment:
In this case it is to warn and not throw error since user asked for a feature and we silently ignored (for now). 

But in previous case, error should be thrown imo to tell user to change derived column config's source column or refresh the segments first to generate forward index on source column and then generate target derivedColumn

Reply:
sure updated the error message in the previous case to indicate next steps for user